inventory_management_system_api/routers/v1/usage_status.py

```diff
@@ -73,3 +78,26 @@ def get_usage_statuses(
 
     usage_statuses = usage_status_service.list()
     return [UsageStatusSchema(**usage_status.model_dump()) for usage_status in usage_statuses]
+
+
+@router.delete(
+    path="/{usage_status_id}",
+    summary="Delete a usage status by its ID",
+    response_description="Usage status deleted successfully",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+def delete_usage_status(usage_status_id: str, usage_status_service: UsageStatusService = Depends()) -> None:
+    # pylint: disable=missing-function-docstring
+    logger.info("Deleting usage status with ID: %s", usage_status_id)
+    try:
+        usage_status_service.delete(usage_status_id)
+    except (MissingRecordError, InvalidObjectIdError) as exc:
+        logger.exception("The specified usage status does not exist")
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND, detail="The specified usage status does not exist"
+        ) from exc
```

Comment:
```suggestion
        message = "Usage status not found"
        logger.exception(message)
        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=message) from exc
```
For consistency with others (it seems manufacturer is the exception).